modules/ROOT/pages/tutorial-user-systemd-unit-on-boot.adoc

```diff
@@ -1,18 +1,84 @@
-There are times when it's helpful to launch a user-level https://www.freedesktop.org/software/systemd/man/systemd.unit.html[systemd unit] without having to log in. For example, you may wish to launch a container that provides a network service or run an HPC job. Below is an example .fcc file that can be converted to ignition for the purpose of launching a systemd service for the user "sleeper". 
+= Launching a user-level systemd unit on boot
+
+NOTE: Make sure that you have completed the steps described in the xref:tutorial-setup.adoc[initial setup page] before starting this tutorial.
+
+In this tutorial, we will setup a user level systemd unit for an unprivileged user. There are times when it's helpful to launch a user-level https://www.freedesktop.org/software/systemd/man/systemd.unit.html[systemd unit] without having to log in. For example, you may wish to launch a container that provides a network service or run an HPC job. For this setup, we will add the following to a Fedora CoreOS config:
+
+* A user level systemd unit: `/home/sleeper/.config/systemd/user/linger-example.service`.
+* Enable it as a user level systemd service.
+
+== Setting up the systemd unit
+
+In this example, we will launch a systemd service for the user "sleeper". First, we add the systemd unit:
+
+[source,yaml]
+----
+storage:
+  files:
+    - path: /home/sleeper/.config/systemd/user/linger-example.service
+      mode: 0744
+      contents:
+        inline: |
+          [Unit]
+          Description=A systemd user unit demo
+          After=network-online.target
+          Wants=network-online.target
+          [Service]
+          ExecStart=/usr/bin/sleep infinity
+      user:
+        name: sleeper
+      group:
+        name: sleeper
+----
+
+System services can be directly enabled in FCC but user level services have to be manually enabled for now:
+
+[source,yaml]
+----
+storage:
+  directories:
+    - path: /home/sleeper/.config/systemd/user/default.target.wants
+      mode: 0755
+      user:
+        name: sleeper
+      group:
+        name: sleeper
+  links:
+    - path: /home/sleeper/.config/systemd/user/default.target.wants/linger-example.service
+      user:
+        name: sleeper
+      group:
+        name: sleeper
+      target: /home/sleeper/.config/systemd/user/linger-example.service
+      hard: false
+----
+
+And finally setup lingering for the systemd user level instance so that it gets started directly on boot and stays running:
+
+[source,yaml]
+----
+storage:
+  files:
+    - path: /var/lib/systemd/linger/sleeper
+      mode: 0644
+----
+
+== Writing the Fedora CoreOS config and converting to Ignition
+
+To make it simpler to log in under each user, we will use SSH keys. The final Fedora CoreOS config, stored in `fcct-user.yaml`, will be:
 
 [source,yaml]
 ----
 variant: fcos
-version: 1.2.0
+version: 1.3.0
 passwd:
   users:
     - name: sleeper
       ssh_authorized_keys:
-        - ssh-rsa ... 
+        - ssh-rsa ...
```

Comment:
I think this can be dropped. Only the interactive admin/debugging user (`core`) would need an SSH key.

Reply:
In this example you need it to access the user and test that the services have been properly started. But I agree this is not ideal.